stubs/bleach/@tests/stubtest_allowlist.txt

```diff
@@ -1,2 +1,6 @@
-bleach.css_sanitizer  # Requires tinycss2 to be installed
-bleach.html5lib_shim.*
+# Internal private stuff:
+bleach._vendor.*
+
+# Hacks:
```

Comment:
@Avasam I don't think you added this comment, but can we say anything more here? This is... not informative :)